tools/mo/openvino/tools/mo/front/tf/conv_ext.py

```diff
@@ -16,9 +16,17 @@ class Conv2DFrontExtractor(FrontExtractorOp):
     @classmethod
     def extract(cls, node):
         attrs = tf_create_attrs(node, 2, 3)
+
+        def get_num_groups(node):
+            num_groups = node.group if 'group' in node else None
+            if num_groups is None and is_fully_defined(node.in_node(0).shape) and is_fully_defined(node.kernel_shape):
+                num_groups = node.in_node(0).shape[node.channel_dims] // node.kernel_shape[node.input_feature_channel]
+            else:
+                num_groups = 1
```

Comment:
it is not correct. Looks a gap

Reply:
Thanks for noticing wrongness of this else. As we discussed I corrected implementation. Now we check not `is_fully_dynamic` but only channel dimensions. Also clarified this `if-elif` statement.

Firstly 
- we try to get number of groups from attribute 
- If attribute is not set  check whether channel dimensions are static and calculate from channel sizes
- and finally if channel sizes are not defined/dynamic set into default num_groups=1

Added corresponding comment into code